client/src/components/SudoModePasswordField/tests/SudoModePasswordField-test.js

```diff
@@ -16,11 +16,29 @@ window.ss.config = {
 };
 
 let doResolve;
+let doReject;
+
+beforeEach(() => {
+  doResolve = undefined;
+  doReject = undefined;
+});
+
+function createJsonError(message) {
```

Comment:
Copied from approach used in https://github.com/silverstripe/silverstripe-elemental/blob/6/client/src/components/ElementActions/tests/ArchiveAction-test.js#L82